scriptworker.yaml.tmpl

```diff
@@ -93,6 +93,19 @@ gpg_path: gpg
 my_email: "scriptworker@example.com"
 
 
+#-----------------------------------------------------------------------------------------------
+# ed25519 settings.
+#-----------------------------------------------------------------------------------------------
+ed25519_private_key_path: /tmp/ed25519_privkey.pem
+ed25519_public_key_paths:
```

Comment:
Hm. These public keys are tiny. For example, the scriptworker throwaway test public key is `KxYrV3XAJ3uOyAUX0Wcl1Oeu6GSMrI/5hOn39q8Lf0I=`. I wonder if we shouldn't just bake these into the config, rather than create plaintext files for each.

Reply:
I'm currently staying with files, because we're less likely to log the contents of a private key file than the running config. When we're verifying these signatures in more places than just scriptworker, it might make sense to publish some json or something that contains all the valid pubkeys.

Reply:
Per IRC, we're leaving private keys in files and putting public keys in config.
In the future, we may move the public keys into their own json file if/when we share the public keys across multiple points of verification.